prov/gni/src/gnix_ep.c

```diff
@@ -2063,26 +2063,44 @@ DIRECT_FN int gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
 	*ep = &ep_priv->ep_fid;
 	return ret;
 
+	int err_ret;
```

Comment:
Shouldn't this declaration be at the start of the function per the style guidelines?